src/webui/www/translations/webui_en.ts

```diff
@@ -2055,7 +2055,7 @@ Use ';' to split multiple entries. Can use wildcard '*'.</source>
 <context>
     <name>PeerListWidget</name>
     <message>
-        <source>IP</source>
+        <source>IP/Address</source>
```

Comment:
No need to make change to this file (or any translation files). We'll take care of it. Please revert this change.